lib/web/mage/menu.js

```diff
@@ -7,7 +7,6 @@ define([
     'jquery',
     'matchMedia',
     'jquery-ui-modules/menu',
-    'jquery/jquery.mobile.custom',
```

Comment:
this file contains "swipeleft" event, that was added in jquery mobile if I'm not mistaken. Is it works now?

Reply:
i have eliminated it already